### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by r0h4n** [comment]
Please file an issue explaining the problem

**Comment by anmolbabu** [comment]
@r0h4n Done

File: tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py [hunk 1]
```diff
@@ -90,10 +98,12 @@ def run(self):
...
-        plugin_config_success &= self._configure_plugin(
-            'tendrl_%s' % NS.tendrl_context.sds_name,
-            plugin_params
-        )
+        if NS.tendrl_context.sds_name == 'gluster':
+            for gluster_plugin in GLUSTER_CLUSTER_PLUGINS:
+                plugin_config_success &= self._configure_plugin(
+                    gluster_plugin,
+                    plugin_params
+                )
```
Comment (r0h4n) [inline tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py]: So you were missing on configuring many plugins before this PR it seems from the code change?

Comment (nthomas-redhat) [inline tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py]: Seems like he had plugin framework avoids this config wherein it picks up everything dropped into a specific folder. Is that right @anmolbabu

**Pull request merged**